pixeltable/share/publish.py

```diff
@@ -102,5 +92,61 @@ def _upload_bundle_to_s3(bundle: Path, parsed_location: urllib.parse.ParseResult
         file=sys.stdout,
     )
     s3_client.upload_file(
-        Filename=str(bundle), Bucket=bucket, Key=str(remote_path), ExtraArgs=upload_args, Callback=progress_bar.update
+        Filename=str(bundle), Bucket=bucket, Key=remote_path, ExtraArgs=upload_args, Callback=progress_bar.update
+    )
+
+
+def pull_replica(dest_path: str, src_tbl_uri: str) -> pxt.Table:
+    headers_json = {'X-api-key': Env.get().pxt_api_key, 'Content-Type': 'application/json'}
+    clone_request_json = {'operation_type': 'clone_snapshot', 'table_uri': src_tbl_uri}
```

Comment:
Leave todo to streamline terminology? We shouldn't have multiple ways to refer to the same thing.